source/api_cc/CMakeLists.txt

```diff
@@ -15,11 +15,20 @@ set (libname "${LIB_DEEPMD_CC}${variant_name}")
 
 add_library(${libname} SHARED ${LIB_SRC})
 
-# link: libdeepmd libdeepmd_op libtensorflow_cc libtensorflow_framework
-target_link_libraries (${libname} PUBLIC ${LIB_DEEPMD}	${TensorFlow_LIBRARY} ${TensorFlowFramework_LIBRARY})
+if (USE_ASCEND)
+  target_link_libraries (${libname} PUBLIC ${LIB_DEEPMD} 
+    ${TensorFlow_LIBRARY} 
+    ${TensorFlowFramework_LIBRARY}
+    ${PY_LIBRARY}
+    dl)
```

Comment:
You should write in a new line when you want to link an additional library.

Reply:
Thanks, we have fixed that in a new line.